test/test_loaders.py

```diff
@@ -38,6 +40,27 @@ def test_base_url_override(openapi3_schema_url, url):
     assert endpoint.base_url == "http://example.com"
 
 
+def test_port_override(openapi3_schema_url):
+    schema = schemathesis.from_uri(openapi3_schema_url, port=8081)
+    endpoint = next(schema.get_all_endpoints())
+    assert endpoint.base_url == "http://127.0.0.1:8081/schema.yaml"
+
+
+def test_port_override_with_ipv6(openapi3_schema_url, simple_openapi, mocker):
```

Comment:
not sure if `simple_openapi` is the payload to mock in `requests.get` - it appears L60 will fail with

```
>           raise InvalidSchema("Schema parsing failed. Please check your schema.") from exc
E           schemathesis.exceptions.InvalidSchema: Schema parsing failed. Please check your schema.
```

Any suggestion, or I can go through the schema code in greater details.